bazel/deps.bzl

```diff
@@ -0,0 +1,71 @@
+"""Dependency specific initialization."""
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_dependencies")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+_ALL_CONTENT = """\
+filegroup(
+    name = "all",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+"""
+
+def deps(repo_mapping = {}):
+    rules_foreign_cc_dependencies()
+
+    maybe(
+        http_archive,
+        name = "rules_foreign_cc",
+        url = "https://github.com/bazelbuild/rules_foreign_cc/archive/refs/tags/0.8.0.tar.gz",
```

Comment:
Maybe add a comment: why 0.8.0? I presume it was the latest that was available at the time we checked (e.g. on 2022/07/07).

Same for the other dependencies below: are these version numbers important, or are they just the latest available?

Reply:
Yep, I did it just to have the latest version. That's it.